docs/src/main/sphinx/release/release-463.md

```diff
@@ -0,0 +1,27 @@
+# Release 463 (23 Oct 2024)
+
+## General
+
+* Add {func}`timezone` functions to extract the timezone identifier from a
+  timestamp or time. ({issue}`20893`)
+* Add table functions to the `SHOW FUNCTIONS` output. ({issue}`12550`)
```

Comment:
"List table functions with `SHOW FUNCTIONS`"

Reply:
yeah .. but thats kinda misleading .. its also listing table functions..

Reply:
Include table functions with SHOW FUNCTIONS output. ??

Reply:
Well, it doesn’t say “list ONLY table functions…”, it just implies that the command can now list table functions. But fair enough. The latter phrasing you suggested works, too.